docs/example-scenario/apps/internal-apim-ase-scenario.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Publishing internal APIs to external consumers
+titleSuffix: Azure Example Scenarios
+description: Use Azure API Management, to modernize and expose intranet legacy web APIs.
+author: ssarwa
+ms.date: 03/12/2019
+ms.topic: example-scenario
+ms.service: architecture-center
+ms.subservice: example-scenario
+ms.custom: fasttrack-new
+social_image_url: ./media/architecture-internal-apim-ase-scenario.jpg
+---
+
+# Publishing internal APIs to external consumers
+
+In this scenario an organization has hosted multiple APIs using an [App Service Environments][ase] and would like to consolidate these APIs internally using [Azure API Management (APIM)][apim] deployed into a Virtual Network. The internal API Management instance could also be exposed to external users to allow for utilization of the full potential of the APIs. This external exposure could be achieved using an [Application Gateways][appgtwy] forwarding requests to the internal API Management service which in turn consumes the apis deployed in the ASE. The most interesting part is that since both the App Service Env and APIM services are integrated in a VNET, having them working together presented new challenges. This required leveraging policies within APIM to add a HOST header to the request as it passed through the APIM service to ensure that the ASE's load balancer would properly forward the request.
+
+This example demonstrates how to configure an API Management Service deployed into a vnet which consumes APIs hosted in an App Service Environments on a secured private VNET and assumes
+- The development team will modernize the APIs back end, which is hosted on WebApps in App Service Environments internally
+- The API Management service is maintained by Administrator/s within the Organizations
+
+## Architecture
+
+![Architecture diagram][architecture]
```

Comment:
Can you describe the traffic flow here and add numbers to the diagram for each step?  There's a lot going on in the diagram but not a lot of explanation of why things are there or how the data is flowing through it.

Reply:
Do u have an example from other solutions. I will try to change accordingly?